.ci/e2e-expected/ddl-batching.txt

```diff
@@ -0,0 +1,8 @@
+       List of relations
```

Comment:
It seems that ci/e2e-psql-v* tests failed. I'm not sure if it's due to this change or something else changed. Can you debug what was the issue?

Reply:
They are fixed now.